packages/flutter_tools/lib/src/commands/build_windows.dart

```diff
@@ -65,6 +67,16 @@ class BuildWindowsCommand extends BuildSubCommand {
         flutterUsage: globals.flutterUsage,
       ),
     );
+
+    // We don't print a size because the containing directory may contain
+    // additional files not seen by the user and because the binary is not
+    // self-contained.
```

Comment:
Consider:

```suggestion
    // We don't print a size because the output directory can contain
    // optional files not needed by the user and because the binary is not
    // self-contained.
```